hardhat.config.js

```diff
@@ -0,0 +1,16 @@
+require('@nomiclabs/hardhat-ethers')
+require('dotenv').config()
+
+module.exports = {
+  defaultNetwork: 'hardhat',
+  networks: {
+    hardhat: {
+      blockGasLimit: 150_000_000,
```

Comment:
this is bc 10x block gas limit for??

Reply:
Actually this isn't needed since we don't do the static calls in the tests. This was copy pasted from another project where I was doing huge static calls. Will remove